config/locales/en.yml

```diff
@@ -361,10 +362,14 @@ en:
     destroy:
       removed_notice: "%{owner_name} was removed from the owners successfully"
       failed_notice: Can't remove the only owner of the gem
-    mfa_required: The gem has MFA requirement enabled, please setup MFA on your account.
+    mfa_required: The gem has multi-factor authentication requirement enabled, please setup multi-factor authentication on your account.
   settings:
     edit:
       title: Edit settings
+      webauthn_credentials: Security device
```

Comment:
I just realized NPM and PyPi both settled on security key instead of security device so i'm going to change this to be consistent with the other package managers

Reply:
👍  Out of curiosity, where have you seen that PyPi settled on Security key? I still see the term Security device on their site. Personally, I prefer "device" since "key" gives a perception that only hardware keys can work but that might be just me 😅

Reply:
oh really? I was looking at the [internal implementation ](https://github.com/pypi/warehouse/pull/5795/files#diff-4d7e23e1c19fbedae06550a287ef846d8f8efe27ee3c54f1a7ec3f4f8b9c5af6R251) and [this public doc](https://pypi.org/project/webauthn/), though in fairness the doc does mention 'security key' as a separate thing from  like touch id, for example, which kind of supports your point

Reply:
Oh gotcha, I was looking at the account pages in PyPI (ie. `https://pypi.org/manage/account/`)
<details>
<img width="880" alt="Screen Shot 2022-09-19 at 2 30 45 PM" src="https://user-images.githubusercontent.com/42748004/191089730-1dc602d1-fe9f-4fe0-853a-d1c1438f24b6.png">
</details>

Reply:
I'm in favour of keeping "security device", since webauthn can use biometric auth on a phone or laptop.

Reply:
Jacques also votes for "security device", so majority wins! I'll leave this as is.